src/System.Security.Cryptography.OpenSsl/src/System/Security/Cryptography/ECCharacteristic2Curve.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Security.Cryptography
+{
+    /// <summary>
+    /// A prime elliptical curve.
+    /// </summary>
+    public sealed class ECCharacteristic2Curve : ECExplicitCurve
+    {
+        public byte[] M;
```

Comment:
Is this a packed form of binomial, trinomial, and pentanomial?  Or is this just binomial?